### PR TITLE
Fix tests, useradm CLI is now useradm-enterprise

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

Fixes: https://gitlab.com/Northern.tech/Mender/mender-helm/-/jobs/1965379733

### CODE REVIEW

File: tests/test-002-create-tenant-and-user-and-login.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ echo "> Creating a new tenant: $TENANT_NAME"
 TENANT_ID=$(kubectl exec $TENANTADM -- tenantadm create-org --name $TENANT_NAME --username "$ADMIN_USERNAME" --password "$ADMIN_PASSWORD" --plan "enterprise")
 
 echo "> Creating a new user for the tenant: $USER_USERNAME / $USER_PASSWORD"
-kubectl exec $USERADM -- useradm create-user --username "$USER_USERNAME" --password "$USER_PASSWORD" --tenant-id "$TENANT_ID" --roles "RBAC_ROLE_PERMIT_ALL"
+kubectl exec $USERADM -- useradm-enterprise create-user --username "$USER_USERNAME" --password "$USER_PASSWORD" --tenant-id "$TENANT_ID" --roles "RBAC_ROLE_PERMIT_ALL"
```
Comment (merlin-northern) [inline tests/test-002-create-tenant-and-user-and-login.sh:20]: we need to correct [readme](https://github.com/mendersoftware/mender-helm/blob/master/README.md#create-a-tenant-and-a-user-from-command-line) as well

Comment (tranchitella) [reply tests/test-002-create-tenant-and-user-and-login.sh:20]: Right, thanks a lot! I force-pushed.

File: README.md [hunk 2]
```diff
@@ -571,7 +571,7 @@ $ tenantadm create-org --name demo --username "admin@mender.io" --password "admi
 You can create additional useres from the command line of the `useradm` pod:
 
 ```bash
-$ useradm create-user --username "demo@mender.io" --password "demodemo" --tenant-id "5dcd71624143b30050e63bed"
+$ useradm-enterprise create-user --username "demo@mender.io" --password "demodemo" --tenant-id "5dcd71624143b30050e63bed"
```
Comment (merlin-northern) [inline README.md:574]: one more below

Comment (tranchitella) [reply README.md:574]: The one below is for open source.

Comment (merlin-northern) [reply README.md:574]: sorry -- discussed over slack. you are right.

**Review by merlin-northern — APPROVED** [review]
lgtm

**Pull request merged**